app/views/devise/mailer/reset_password_instructions.html.slim

```diff
@@ -0,0 +1,7 @@
+.row
+  .col-xs-12.col-md-8.col-md-offset-2
+    p Hello = @resource.email!
```

Comment:
I think you want `p Hello #{@resource.email}!`.
Alternatively, you could split it into separate lines like this:
```
p
  |> Hello
  = @resource.email
  | !
```

The pipe character tells slim that this line is just text instead of a HTML tag and the > character tells it to add a space at the end.